release/license_info.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Copyright (C) 2018 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# shellcheck source=common.sh
+
+set -e
+
+SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
+. "${SCRIPT_DIR}/../common.sh"
+. ./config/config.sh
+
+var_load MIX_VERSION
+
+generate_license_info() {
+    pushd "${MIXER_DIR}" > /dev/null
+
+    local tmpfile1=$(mktemp -p .)
+    awk '{print $1}' < update/image/${MIX_VERSION}/os-packages |
+        sort -u |
+        xargs dnf -q -c .yum-mix.conf --releasever="$(< upstreamversion)" --installroot=dnf-cache repoquery --queryformat "%{LICENSE}" |
+        tr ' ' '\n' |
+        sort -u > "$tmpfile1"
+
+    # Filter out non-license words
+    local blacklist="and"
+    local tmpfile2=$(mktemp -p .)
+    for word in $blacklist; do
+        awk "\$0 != \"$word\" {print}" "$tmpfile1" > "$tmpfile2"
```

Comment:
instead of 'awk' and keep moving files every iteration just do a sed in place every iteration. It yields a cleaner code:

```bash
sed -e "/^${word}$/d" -i "${tmpfile1}"
```
That + moving to the work dir even makes the use of temp files unnecessary:

```bash
file="${WORK_DIR}/${PKG_LICENSES_FILE}"
: > "${file}"

...
sort -u > "${file}"
....
sed ... -i "${file}"
```

you dont even need 'pushd/popd' in this case.